.github/workflows/run-snapshot-generation.yml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
thought: maybe we could include in readme the information that if you add `[New snapshots]` the screenshots are recorded 🤔

Reply:
Done that 👌

Reply:
Ahh, in a readme. I thought about adding a comment in a yaml file. In readme it is even a better idea. 👌